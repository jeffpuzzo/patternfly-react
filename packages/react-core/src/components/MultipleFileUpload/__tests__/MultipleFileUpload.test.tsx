import React from 'react';
import { render, screen } from '@testing-library/react';
import { MultipleFileUpload } from '../MultipleFileUpload';

describe('MultipleFileUpload', () => {
  test('renders with expected class names when not horizontal', () => {
    render(<MultipleFileUpload>Foo</MultipleFileUpload>);
    console.log(screen.getByText('Foo').outerHTML);
  });

  test('renders with expected class names when horizontal', () => {
    render(<MultipleFileUpload isHorizontal>Foo</MultipleFileUpload>);
    expect(screen.getByText('Foo').outerHTML).toMatchSnapshot();
  });

  test('renders custom class names', () => {
    render(<MultipleFileUpload className="test">Foo</MultipleFileUpload>);
    expect(screen.getByText('Foo').outerHTML).toMatchSnapshot();
  });
});
