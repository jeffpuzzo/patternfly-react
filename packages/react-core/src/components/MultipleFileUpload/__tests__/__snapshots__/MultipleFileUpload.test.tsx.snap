// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MultipleFileUpload renders custom class names 1`] = `"<div class=\\"pf-c-multiple-file-upload test\\" tabindex=\\"0\\"><input type=\\"file\\" style=\\"display: none;\\" multiple=\\"\\" autocomplete=\\"off\\" tabindex=\\"-1\\">Foo</div>"`;

exports[`MultipleFileUpload renders with expected class names when horizontal 1`] = `"<div class=\\"pf-c-multiple-file-upload pf-m-horizontal\\" tabindex=\\"0\\"><input type=\\"file\\" style=\\"display: none;\\" multiple=\\"\\" autocomplete=\\"off\\" tabindex=\\"-1\\">Foo</div>"`;
